Demo/Demo/SwiftUIComponents/CardVaultViews/CardVaultView.swift

```diff
@@ -0,0 +1,71 @@
+import SwiftUI
+
+struct CardVaultView: View {
+
+    @StateObject var baseViewModel = BaseViewModel()
+    @StateObject var cardVaultViewModel = CardVaultViewModel()
+
+    private let cardFormatter = CardFormatter()
+
+    // MARK: Views
+
+    var body: some View {
+        ScrollView {
+            ScrollViewReader { scrollView in
+                VStack(spacing: 16) {
+                    CreateSetupTokenView(baseViewModel: baseViewModel, cardVaultViewModel: cardVaultViewModel)
+                    if let setupTokenResponse = cardVaultViewModel.state.setupTokenResponse {
+                        SetupTokenResultView(setupTokenResponse: setupTokenResponse)
+                        let setupToken = setupTokenResponse.id
+                        UpdateSetupTokenView(baseViewModel: baseViewModel, cardVaultViewModel: cardVaultViewModel, setupToken: setupToken)
+                            .id("updateSetupTokenView")
+                    }
+                    if let updateSetupTokenResponse = cardVaultViewModel.state.updateSetupToken {
+                        let setupToken = updateSetupTokenResponse.id
+                        UpdateSetupTokenResultView(updateSetupTokenResponse: updateSetupTokenResponse)
+                        CreatePaymentTokenView(baseViewModel: baseViewModel, cardVaultViewModel: cardVaultViewModel, setupToken: setupToken)
+                            .id("createPaymentTokenView")
+                    }
+                    if let paymentTokenResponse = cardVaultViewModel.state.paymentTokenResponse {
+                        PaymentTokenResultView(paymentTokenResponse: paymentTokenResponse)
+                            .id("paymentTokenResultView")
+                        Button("Reset") {
+                            cardVaultViewModel.resetState()
+                        }
+                        .foregroundColor(.white)
+                        .padding()
+                        .frame(maxWidth: .infinity)
```

Comment:
qq: What does maxWidth `.infinity` do? I'm n 👀 b status with SwiftUI.

Reply:
It expands the view as much as possible to its container view with the padding. Without it, sometimes you get it just big enough for the content the view contains.